### ISSUE TITLE
failure of rp 0.3.14 gromacs/lsdmap on archer

### ISSUE THREAD

**Issue by ebreitmo** [radical-cybertools/ExTASY#234]
virtualenv-1.11.6/scripts/virtualenv 14Jan
source 14Jan/bin/activate
pip install radical.ensemblemd
ensemblemd-version
0.3.14
mkdir $HOME/extasy-tutorial/
cd $HOME/extasy-tutorial/

(Following instructions from  http://extasy.readthedocs.org/en/latest/pages/grlsd.html )

curl -k -O  https://raw.githubusercontent.com/radical-cybertools/ExTASY/extasy_data/tarballs/grlsd-on-archer.tar.gz
tar xvfz grlsd-on-archer.tar.gz
cd grlsd-on-archer
emacs -nw archer.rcfg (changing username to ‘ebreitmo’)

python extasy_gromacs_lsdmap.py --RPconfig archer.rcfg --Kconfig gromacslsdmap.wcfg

================================================================================
 EnsembleMD (0.3.14)                                                            
================================================================================

Starting Allocation                                                           ok
        Verifying pattern                                                     ok
        Starting pattern execution                                            ok
--------------------------------------------------------------------------------
Executing simulation-analysis loop with 1 iterations on 24 allocated core(s) on 'epsrc.archer'

Job waiting on queue...
Job is now running !
pre_loop() not defined. Skipping.2016-01-14 16:53:41,117: radical.enmd.SingleClusterEnvironment: MainProcess                     : MainThread     : ERROR   : Fatal error during execution: global name 'grompp_options' is not defined.
Fatal error during execution: global name 'grompp_options' is not defined.
        Starting Deallocation2016-01-14 16:53:41,117: radical.enmd.SingleClusterEnvironment: MainProcess                     : MainThread     : ERROR   : Fatal error during execution: global name 'grompp_options' is not defined.
Fatal error: global name 'grompp_options' is not defined.  File "/Users/elenabreitmoser/14Jan/lib/python2.7/site-packages/radical/ensemblemd/single_cluster_environment.py", line 244, in run
    plugin.execute_pattern(pattern, self)
  File "/Users/elenabreitmoser/14Jan/lib/python2.7/site-packages/radical/ensemblemd/exec_plugins/simulation_analysis_loop/static.py", line 250, in execute_pattern
    if isinstance(pattern.simulation_step(iteration=iteration, instance=1),list):
  File "extasy_gromacs_lsdmap.py", line 93, in simulation_step
    if grompp_options is not None:

        done

**Comment by vivek-bala** [comment]
This should be resolved in master and extasy_data. I tested on stampede. Please try again.

**Comment by vivek-bala** [comment]
similar to #232

**Comment by ibethune** [comment]
OK, so this problem is indeed fixed if you install from master.  I am now hitting a different problem (new issue to follow...)

**Comment by ibethune** [comment]
Elena, if the default grlsd workflow is working for you, please close this.  Having said that, I'm hitting #226
